example/cumulus-tf/variables.tf

```diff
@@ -441,3 +441,7 @@ variable "default_tags" {
   }
 }
 
+variable "optional_dynamo_tables" {
```

Comment:
It's not clear to me what this is being used for (obviously it adds additional dynamo tables that aren't being managed by the data-persistence module, but why would that situation exist?).    Consider a description here.

Reply:
[`673994a` (#2165)](https://github.com/nasa/cumulus/pull/2165/commits/673994a2152c8d4ce29465202ea7610be01d05ff) 
Let me know what you think of that description!

Reply:
For clarity, I added this variable so that I could avoid having to continually manually update my lambdas to read from the `podaac-uat` tables for testing.

Reply:
That description looks good!